src/UI/Component/Panel/Listing/Factory.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Interface Factory
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Factory {
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      The Appointment Listing Panel is used to display a list of
```

Comment:
we are in ILIAS\UI\Component\Panel\Listing, use the Factory::appointment() and get? An Appointment. But: What we really got was a AppointmentListingPanel. In my opinion the problem is a miss with the name:
- Appointment -> AppointmentListingPanel
- AppointmentItem -> This should be the Appointment

Have a look at the doc: 
`@param AppointmentItem[] $items Set of Appointements to be displayed`: In this sentence the conflicts is already written, It's talking about Appointements but wants AppointmentItems
`*      The Appointment Listing Panel is used to displayed a list of`: this is naming the thing correctly. I know this is the discussion about FQN with or without namespace. But the example shows how one can get confused when using the UI-Service and gets an Appointment when he wants an AppointmentListingPanel. 



Besides this: there is currently no Interface for AppointmentItem